pages/index.tsx

```diff
@@ -29,14 +29,14 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
   return (
     <Page className="flex align-items-center">
       <div className="flex-1 width-full">
-        <section className="padding-top-4xl padding-bottom-xl">
+        <section className="padding-top-4xl padding-bottom-l">
           <PageContainer>
```

Comment:
I'm not sure we want to assume flex on all page containers - we can achieve the intended style with a class name 
for this specific case:
```suggestion
          <PageContainer className="flex align-items-stretch">
```

If we want flex he

Reply:
I was playing around with aligning the dropdown to the right and although I didn't actually do that, when we introduce sort, having this as flex will enable that

Reply:
Seems unnecessary to have an additional set of classes when we already have a class for this one instance of page-container? Can always abstract it later if we find we use page-container where we don't need flex?

Reply:
This isn't the one instance of page container though? - there should be 3 on the home page alone (seems the footer one got dropped somewhere along the way!), and in the case we add sort they'll all have different styling - which is fine. 

I don't really see this as "additional classes" but styling this differently to the other cases of page container - it's like a variant of the component. The way we've decided to do that is to using the className, but it's the equivalent of creating props on the component. 

I think opt out is way more confusing than opt in to be honest - I like adding the additional classes (even if it looks unnecessary!) because it's really explicit about what this component does and how it's styled. If we make it flex by default, we'll end up in the situation where we do: `<PageContainer className="align-items-center" />` which kind of doesn't make sense imo (because the flex is missing)